captum/optim/_core/objectives.py

```diff
@@ -196,6 +201,45 @@ def loss_function(targets_to_values: ModuleOutputMapping):
     return loss_function
 
 
+def deepdream(target: nn.Module) -> LossFunction:
+    def loss_function(targets_to_values: ModuleOutputMapping):
+        activations = targets_to_values[target]
+        return activations ** 2
+
+    return loss_function
+
+
+def l1(target: nn.Module, constant: float = 0) -> LossFunction:
+    def loss_function(targets_to_values: ModuleOutputMapping):
+        activations = targets_to_values[target]
+        return torch.abs(activations - constant)
+
+    return loss_function
+
+
+def diversity(target: nn.Module) -> LossFunction:
+    def loss_function(targets_to_values: ModuleOutputMapping):
```

Comment:
> Having CV transformers similar to torchvision would be great. It would be good to rely on torchvision if there are equivalent transformers available in torchvision package so that we do not have to reimplement the same thing. Which transformers are you exactly talking about ?

Right now `center_crop` is a function, whereas the main PIL image equivalent in Torchvision is a class. Torchvision also has a RandomAffine class, and each random affine transformation has it's own class for use individually. There's also a functional version of each transform in Torchvision.

I also thought we were trying to avoid requiring Torchvision?

Reply:
Okay, I've added a short description, a citation, and a link for the Distill article that it comes from!